src/base58.h

```diff
@@ -14,6 +14,8 @@
 #ifndef BITCOIN_BASE58_H
 #define BITCOIN_BASE58_H
 
+#include <attributes.h>
```

Comment:
Same here

Reply:
`attributes.h` is correct but that file got lost in the last rebase. Now re-added :-)

Reply:
Seems `attributes.h` is as same as `nodiscard.h` now

Reply:
`nodiscard.h` now removed :-)